packages/frontend/src/components/ExportCSV/index.tsx

```diff
@@ -79,6 +79,7 @@ const ExportCSV: FC<ExportCSVProps> = memo(
                         pollCsvFileUrl(scheduledCsvResponse)
                             .then((csvFile) => {
                                 if (csvFile.url)
+                                    // eslint-disable-next-line react-compiler/react-compiler
                                     window.location.href = csvFile.url;
```

Comment:
wonder if we can do something else

error is:  Writing to a variable defined outside a component or hook is not allowed. Consider using an effect

ref issue: https://github.com/facebook/react/issues/31544

Reply:
more relevant issue: https://github.com/facebook/react/issues/31264#issuecomment-2552072748